.circleci/config.yml

```diff
@@ -38,11 +38,26 @@ jobs:
           command: |
               xcodebuild -destination 'platform=tvOS Simulator,name=Apple TV 4K,OS=latest' -sdk appletvsimulator -scheme "Fetch-tvOS" clean test |
               tee xcodebuild.log
-
+  macOStest:
+    macos:
+      xcode: "9.3.0"
```

Comment:
Is it possible to have variables in the circle yml file? Would be good to extract this if possible.

Reply:
We don't unfortunately, I got rid of mine...